primitives/core.fil

```diff
@@ -1,2 +1,10 @@
 import "./comb.fil";
-import "./state.fil";
\ No newline at end of file
+import "./state.fil";
+
+extern "core.sv" {
+   comp Wire[WIDTH]<'G, 'L>(
```

Comment:
What would this be for? I imagine it would be for renaming ports, but it can probably be done with Filament bundles.

If you have some port `p0`, I guess you would want to pass it to some instantiation `add_in := new Wire[W]<'G>(p0)` to get a port named `add_in.out` that would be passed into an adder?

If you have `p0`, you can define a bundle `bundle add_in[1]: 'G, 'G+1] W` and then make the assignment `add_in{0} = p0`. Then you'd use `add_in{0}` wherever. To me, this is also more readable.

Reply:
It makes implementing Spade's `Alias` operation easier.

Reply:
I think we should move Spade-specific stuff to a `spade.fil` file instead instead of adding things to the standard library.

Reply:
Good idea, I was thinking that too